netebpfext/net_ebpf_ext.c

```diff
@@ -28,11 +28,34 @@ NDIS_HANDLE _net_ebpf_ext_ndis_handle = NULL;
 NDIS_HANDLE _net_ebpf_ext_nbl_pool_handle = NULL;
 HANDLE _net_ebpf_ext_l2_injection_handle = NULL;
 
+static net_ebpf_ext_sublayer_info_t _net_ebpf_ext_sublayers[] = {
+    {
+        &EBPF_DEFAULT_SUBLAYER,
+        L"EBPF Sub-Layer",
+        L"Sub-Layer for use by eBPF callouts",
+        0,
+        FWP_EMPTY // Auto weight.
+    },
+    {
+        &EBPF_HOOK_ALE_CONNECT_REDIRECT_V4_SUBLAYER,
```

Comment:
I would rename these sublayers to match the ebpf attach_type. So this is the sublayer for cgroup_connect_v4 attach type etc.

Reply:
Done